report/reportFiles/script.js

```diff
@@ -0,0 +1,189 @@
+function filterTarget() {
+    var input, filter, table, tr, td, i, txtValue;
+    input = document.getElementById("myInput");
+
+    filter = input.value.toUpperCase().split(" ").join("_");
+    table = document.getElementById("statusTable");
+    tr = table.getElementsByTagName("tr");
+
+    if (filter.startsWith('"') && filter.endsWith('"')) {
+        filter = filter.substring(1, filter.length - 1)
+        for (i = 0; i < tr.length; i++) {
+            td = tr[i].getElementsByTagName("td")[0];
+            if (td) {
+                txtValue = td.textContent || td.innerText;
+                if (txtValue.toUpperCase() === filter) {
+                    tr[i].style.display = "";
+                } else {
+                    tr[i].style.display = "none";
+                }
+            }
+        }
+    } else {
+        for (i = 0; i < tr.length; i++) {
+            td = tr[i].getElementsByTagName("td")[0];
+            if (td) {
+                txtValue = td.textContent || td.innerText;
+                if (txtValue.toUpperCase().indexOf(filter) > -1) {
+                    tr[i].style.display = "";
+                } else {
+                    tr[i].style.display = "none";
+                }
+            }
+        }
+    }
+}
+
+//refresh page every 5 minutes
+setTimeout(function () {
+    window.location.reload(1);
+}, 1000 * 5 * 60);
+
+function doRequest() {
+
+    // Creating Our XMLHttpRequest object 
+    var xhr = new XMLHttpRequest();
+
+    // Making our connection  
+    var url = "http://" + window.location.hostname + '/statusReport';
```

Comment:
we had to change is line to var 
`url = "http://localhost:5000/statusReport";`

not sure is it important or not.